### PR TITLE
E UI 4708 court name

### PR DESCRIPTION
**Pull request by waynepHMCTS**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
* Change from site name to court name
*
*

### CODE REVIEW

File: projects/exui-common-lib/src/lib/components/search-location/search-location.component.ts [hunk 1]
```diff
@@ -86,6 +86,6 @@ export class SearchLocationComponent implements OnInit {
   }
 
   public getDisplayName(selectedLocation: LocationModel): string {
-    return selectedLocation.site_name;
+    return selectedLocation.court_name;
```
Comment (rayliangatos) [inline projects/exui-common-lib/src/lib/components/search-location/search-location.component.ts:89]: do we need a separate method to get the court_name, can it be put in html directly?

**Review by rayliangatos — APPROVED** [review]
approved with comment

**Review by Kartheek-S — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "1.4.8-select-all-redo",
+  "version": "1.4.12-court-nam",
```
Comment (Daniel-Lam) [inline package.json:3]: @waynepHMCTS - looks like a typo here... should it not be `-court-name`?

**Review by Daniel-Lam — COMMENTED** [review]
@waynepHMCTS - spotted a possible typo in your version naming.

**Pull request merged**